public/views/mentor.html

```diff
@@ -40,36 +40,60 @@
     </div>
     <div class="row">
       <div class="col-xs-12 col-md-10 col-md-offset-1">
-        <div class="container-fluid the-mentors">
-          <div class="row">
-            <section class="col-xs-12 col-sm-4 mentor-profile">
-              <header class="mentor-header">
-                <img src="/img/img-mentor-emma-erwin.jpg" class="img-circle mentor-face" alt="{{ 'Emma' | i18n }}"/>
+        <h1>{{ 'What is web literacy' | i18n }}</h1>
 
-                <h3>{{ 'Emma' | i18n }}</h3>
-                <h4>{{ 'Video remixing and open source for youth' | i18n }}</h4>
-              </header>
-              <p bind-unsafe-html="'About Emma' | i18n"></p>
-            </section>
-            <section class="col-xs-12 col-sm-4 mentor-profile">
-              <header class="mentor-header">
-                <img src="http://content-2011.mozillafestival.org/uploads/2011/09/jonrogers-sm.jpg" class="img-circle mentor-face" alt="{{ 'Jon' | i18n }}"/>
+        <div class="lead">
```

Comment:
Giving this a class of `lead` seems to throw the page off-balance a little to me:

![](http://snaps.akibraun.com/hfjz8.png)